src/Formatter/Formatter.Tests/Examples.fs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/// <summary>
+/// <see cref="QsFmt.Formatter.Tests.Example"/> test cases.
+/// </summary>
+module Microsoft.Quantum.QsFmt.Formatter.Tests.Examples
+
+[<Example>]
+let ``Indents function`` =
+    """namespace Foo {
+function Bar() : Int {
+let x = 5;
+return x;
+}
+}""",
+
+    """namespace Foo {
+    function Bar() : Int {
+        let x = 5;
+        return x;
+    }
+}"""
+
+[<Example>]
+let ``Indents if-else statement`` =
+    """namespace Foo {
+function Bar() : Int {
+let x = 5;
+if (x == 5) {
+return x;
+} else {
+return x + 1;
+}
+}
+}""",
+
+    """namespace Foo {
+    function Bar() : Int {
+        let x = 5;
+        if (x == 5) {
```

Comment:
Minor question, but do we want the formatter to keep parens here? I thought that parens after `if` were generally discouraged now that they're optional?

Reply:
Yes, that would be a good formatting rule to add. The formatter hasn't been taught to remove parentheses yet, and I think this test case was written before QEP 1 was implemented. I would like to delay new formatting rule additions to later PRs though.